scripts/generate-sitemap.sh

```diff
@@ -12,15 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: Markdown
-on:
-  pull_request:
-    branches:
-      - v0
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: xt0rted/markdownlint-problem-matcher@v1
-      - run: (cd site && npm run lint-check)
+set -o errexit
+set -o pipefail
+
+cd site
+npm i
+
+npx sitemap http://localhost:3000/
```

Comment:
Does it expects site being served at `localhost:3000` ?  `make site-run-server` needs to be run before this script ?

Reply:
Yes, updated the Makefile to run it as part of the command.